src/domain/common/form/form.entity.ts

```diff
@@ -4,9 +4,9 @@ import { IForm } from './form.interface';
 
 @Entity()
 export class Form extends BaseAlkemioEntity implements IForm {
-  @Column('text')
+  @Column('text', { nullable: false })
   questions!: string;
 
   @Column('text', { nullable: true })
-  description = '';
+  description?: string;
```

Comment:
We may want this to be mandatory. And actually later I could imagine that a form also gets a profile if it becomes more widely used. But for now I would suggest mandatory.

Reply:
Agreed that this should then also be enforced on the api layer